### PR TITLE
Add an example for the show-capacity flag

### PR DESCRIPTION
**Pull request by bysnupy**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Add new example for the show-capacity flag. If you specified this flag, show metrics based on capacity which is the total amount of resources that a Node has. You can see further information about capacity at [Capacity and Allocatable](https://kubernetes.io/docs/concepts/architecture/nodes/#capacity).

* References: https://github.com/kubernetes/kubernetes/pull/102917

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by rikatz** [comment]
/ok-to-test
/lgtm

We are on code freeze, so this probably will only enter after it. As it's mostly about example I think it's fine to wait.
Thanks!

File: staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go:71]: Usually we tend to add examples for non-obvious flags, this particular cases doesn't add anything more than what this particular flag description.

**Review by soltysh — CHANGES_REQUESTED** [review]
/hold

File: staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go
Comment (bysnupy) [reply staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go:71]: Thank you for your review. It makes sense for me as well. 
Just I reckoned this example is reasonable compared to `kubectl top pod` examples. Such like `kubectl top pod POD_NAME --containers`. The `--show-capacity` flag makes the output switch to another mode like above `--containers`, so I thought it helps adding this example to `kubectl top node` for more kind example usage. So I'd like to add this example for better user experience.

**Comment by dims** [comment]
this PR has been on hold for about 2 weeks. Can you please see how to unblock this by reviewing comments etc? or please close this out (Trying to declutter our open list of PRs)

File: staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go
Comment (soltysh) [reply staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go:71]: I'm not convinced that we should add examples just to have examples, which is what you're saying.

Comment (bysnupy) [reply staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go:71]: @soltysh Thank you for your review. I got understand you. I also don't want to push this hard. I closed this ticket.

**Pull request closed without merging**
